src/cli/miner_cli_info.erl

```diff
@@ -362,5 +362,10 @@ format_macs_from_interfaces(IFs)->
     [], IFs).
 
 format_hwaddr(HWAddr)->
-    HWAddr0 = [integer_to_list(B,16) || B <- HWAddr ],
+    HWAddr0 = [pad_and_format_int(B) || B <- HWAddr ],
```

Comment:
```suggestion
   blockchain_utils:bin_to_hex(list_to_binary(HWAddr)).
```

Reply:
looks good.  I tested your suggestion locally and does the job